sklearn/tree/_splitter.pyx

```diff
@@ -345,75 +345,77 @@ cdef class BestSplitter(BaseDenseSplitter):
                 features[n_drawn_constants], features[f_j] = features[f_j], features[n_drawn_constants]
 
                 n_drawn_constants += 1
+                continue
```

Comment:
Most of this PR is adding `continue` in a few palces, so there is less indentations from the `else` clause.